apps/dashboard/src/components/primitives/field-editor/variable-plugin/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const VARIABLE_REGEX = /{{([^{}]+)}}/g;
```

Comment:
Nit-picky one: Conside moving this contents to the main file. Three constants don't justify another file that adds an overhead to the live reload and building process.